app/konnect/getting-started/index.md

```diff
@@ -2,60 +2,24 @@
 title: Get started with Konnect
 ---
 
-## Quickstart guide
+New to {{site.konnect_saas}}? You can get started with the basics by following the steps in the [onboarding wizard in the web app](https://konghq.com/products/kong-konnect/register?utm_medium=referral&utm_source=docs&utm_campaign=gateway-konnect&utm_content=gateway-quickstart-install). 
```

Comment:
Here are some options that could also work: 

New to {{site.konnect_saas}}? Use our in-app onboarding wizard for a streamlined setup process. [Start now]()

First time with {{site.konnect_saas}}? Our in-app wizard guides you through the initial setup efficiently. [get started]()